game/static/game/css/board.css

```
@@ -0,0 +1,1135 @@
+    box-shadow: 0 0 0 2px rgba(240, 192, 64, 0.2);
+    outline: none;
+}
+
+.visually-hidden {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.11.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)